dirAlias.js

```diff
@@ -8,6 +8,7 @@ module.exports = {
   webpackDirAlias: {
     '#app': resolvePath('src/app'),
     '#contexts': resolvePath('src/app/contexts'),
+    '#components': resolvePath('src/app/components'),
```

Comment:
I love a good alias 👍

Reply:
Yes, this will become more helpful as and when we migrate more Psammead packages!